Makefile

```diff
@@ -3,20 +3,17 @@
 ###############################
 CLUSTER_GSLB1 = test-gslb1
 CLUSTER_GSLB2 = test-gslb2
+CLUSTER_GSLB_NETWORK = k3d-action-bridge-network
 GSLB_DOMAIN ?= cloud.example.com
 REPO = absaoss/k8gb
 VALUES_YAML ?= chart/k8gb/values.yaml
 PODINFO_IMAGE_REPO ?= stefanprodan/podinfo
 HELM_ARGS ?=
 K8GB_COREDNS_IP ?= kubectl get svc k8gb-coredns -n k8gb -o custom-columns='IP:spec.clusterIP' --no-headers
 ETCD_DEBUG_IMAGE ?= quay.io/coreos/etcd:v3.2.25
-CLUSTER_GSLB2_HELM_ARGS ?= --set k8gb.clusterGeoTag='us' --set k8gb.extGslbClustersGeoTags='eu' --set k8gb.hostAlias.hostnames='{gslb-ns-cloud-example-com-eu.example.com}'
-
 
-# terratest
-GITACTION_TERRATEST_DOCKER_REPO_PORT ?= 5000
-GITACTION_TERRATEST_DOCKER_REPO_NAME ?= kind-registry
-GITACTION_TERRATEST_DOCKER_REPO_IMAGE ?= registry:2
+CLUSTER_GSLB2_HELM_ARGS ?= --set k8gb.clusterGeoTag='us' --set k8gb.extGslbClustersGeoTags='eu' --set k8gb.hostAlias.hostnames='{gslb-ns-cloud-example-com-eu.example.com}'
+GITACTION_IMAGE_TAG ?=registry.localhost:5000/k8gb
```

Comment:
nitpick - it's actually not a tag but a image repo. The tag is commit hash below